packages/react/src/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { clsx } from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+export interface BreadcrumbItemProps extends PropsWithChildren<HTMLAttributes<HTMLLIElement>> {
+  href: string
+}
+
+export const BreadcrumbItem = forwardRef(
+  ({ children, className, href, ...restProps }: BreadcrumbItemProps, ref: ForwardedRef<HTMLLIElement>) => (
+    <li {...restProps} className={clsx('amsterdam-breadcrumb__item', className)} ref={ref}>
+      <a className="amsterdam-breadcrumb__link" href={href}>
+        {children}
+      </a>
+    </li>
+  ),
+)
+
+BreadcrumbItem.displayName = 'BreadcrumbItem'
```

Comment:
We’re not consistent with including the dot here or not. I say we do. (More elsewhere.)

```suggestion
BreadcrumbItem.displayName = 'Breadcrumb.Item'
```

Reply:
We currently define the display name twice, here and in the main component file. That's probably not necessary. 

I made [a new ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-591) to fix some issues with the subcomponent typings, let's resolve that there.